### ISSUE TITLE
[amp-sidebar 1.0][a11y] Add skip nav link

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#10558]
Skipnav links are invisible links that allow keyboard or screen-reader users to jumps over the navigation links and land on the main content.

Be mindful of buggy mobile screenreader behaviour for off-screen clickable targets. Look at how eixisting sidebar adds screenReaderCloseButton and take a similar approach.

We also need to dynamically figure out the target of skip nav. The first `<main>` makes sense but it may not be there. Maybe first element after <amp-sidebar> ends is the right one.

It only makes sense to have this when there is a `navbar`, a collapsed sidebar navigation does not need skip nav.

/cc @camelburrito @torch2424

**Comment by torch2424** [comment]
After discussing with @aghassemi , we decided this would mainly affect the `toolbar` attribute of Sidebar 1.0. The concern is that in the main use case, developers will use toolbar as a top nav bar, and should be able to skip it. It is suggested we add a "skip to toolbar content" element at the beginning of the target. And a focusable element at the end of the target, to enable skipping the content of a target.

My only concerns are that not all toolbars will be used for top navigation bars. We may have multiple toolbars (So multiple "Skip to content would sound strange"), and we may need to expose an attribute to add an aria-label to this skip button, for different languages and developer control

**Comment by ampprojectbot** [comment]
This issue seems to be in  Pending Triage for awhile. @torch2424 Please triage this to an appropriate milestone.

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @torch2424 Do you have any updates?

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @aghassemi Do you have any updates?
